changelog.d/11677.bugfix

```diff
@@ -0,0 +1 @@
+Fix wrong variable reference in `SimpleHttpClient.get_json` that results in the absence of the `Accept` header in the request.
```

Comment:
This isn't a *great* newsfile, on two counts:

* People using synapse don't normally care *how* a bug gets fixed (and if they do, they can click through to the PR). So "wrong variable reference" and "`SimpleHttpClient.get_json`" aren't useful here and we should just omit the first part of the sentence.
* We talk about "the request", but what request? You're assuming that people know that `SimpleHttpClient` is used to make outbound HTTP requests, but most people reading the changelog *don't* know that, and in any case it's not used for all outbound requests. Anyway, what is the impact of omitting an `Accept` header?

Generally it would be great if we could rephrase this in terms of a change in functionality that people will actually see.

@Fr3shTea maybe if you could propose some better wording we can just commit it to the repo.

Reply:
This is a very small change and I'm not sure that it will have significant effect in terms of change in functionality. Maybe something is relying on that `Accept` header, but I'm not sure about that either. I mostly created this newsfile because reivilibre told me to do that and I can't really think of any better wording for that small change.

Reply:
Maybe newsfile might be unnecessary for such fix?

Reply:
generally if we don't think it's going to have a noticeable effect, we give it a `.misc` newsfile so that it appears under "Internal changes".

I've actually realised this has now made it into the changelog for 1.50: https://github.com/matrix-org/synapse/blob/83acdb23fe318f8a41d2a6b800d994d46754a903/CHANGES.md, so it's too late to change the newsfile anyway.

@reivilibre can I hand this over to you to do *something* with in the 1.50 changelog?

Reply:
@richvdh Yup I'll take care of it